codegen/src/main/java/software/amazon/awssdk/codegen/emitters/CodeWriter.java

```diff
@@ -100,22 +99,20 @@ public CodeWriter(String dir, String file, String fileNameSuffix) {
      */
     @Override
     public void flush() {
-        PrintWriter out = null;
         try {
-            File outputFile = Utils.createFile(dir, this.file);
+            File outputFile = new File(dir, this.file);
```

Comment:
nit: use `Paths.get()` so we can remove call to `toPath()` later

Reply:
Fixed.